webapp/client/client.jsx

```diff
@@ -921,6 +921,15 @@ export default class Client {
             end(this.handleResponse.bind(this, 'getUser', success, error));
     }
 
+    getUsername(userName, success, error) {
```

Comment:
same here `getUserByUsername` and probably the first parameter can be `username` instead of `userName`